session/bootstrap.go

```diff
@@ -964,6 +964,9 @@ func getTiDBVar(s Session, name string) (sVal string, isNull bool, e error) {
 // For example, add new system variables into mysql.global_variables table.
 func upgrade(s Session) {
 	ver, err := getBootstrapVersion(s)
+	if err != nil {
+		fmt.Println("ErrorStack:", errors.ErrorStack(err))
```

Comment:
Should use ```log.Error``` ? It looks like a debug code.

Reply:
The debugging log added mistakenly, I'll remove this